.env.local

```diff
@@ -1,6 +1 @@
-DATABASE_URL=postgresql://user:pass@localhost:5432/waect
```

Comment:
I changed this to the lastest version with the current env variables needed for the python flask. Probably u run into merge conflicts here

Reply:
ok